### DISCUSSION TITLE
mongodb insert ISODate

### DISCUSSION THREAD

**Discussion by Valdri** [Budibase/budibase#8711] [Help] [answered]
How to insert ISODate to mongodb time-series collection with bb query?
when i try this:
![image](https://user-images.githubusercontent.com/13934649/202470844-2ed24b7c-3a21-4cb4-b63c-652786c431e4.png)

I get an error:
"Query Error: Cannot create property '_id' on string '{ "timestamp": ISODate('2021-05-18T00:00:00.000Z') }'"

**Comment by melohagan** [comment, marked as answer]
Hey @Valdri 

There is currently no support for date objects in the MongoDB connector. I've raised an issue here: https://github.com/Budibase/budibase/issues/8746
